dts: ra8: fix syntax error in r7fa8XXXX.dtsi files

gurland opened the pull request:
The "status" property was defined after the pllX subnodes in pll node, which violates the Devicetree Specification v0.4, section 6.3:

"Nodes may contain property definitions and/or child node definitions. If both are present, properties shall come before child nodes."

This caused the Device Tree Compiler error: "Properties must precede subnodes. Unable to parse input tree"

This commit moves the "status" property to precede all pll subnodes, fixing the syntax error and ensuring compliance with the Devicetree Specification.

khoa-nguyen-18 reviewed (approved):
The update looks good to me

gurland committed:
dts: ra8: fix syntax error in r7fa8XXXX.dtsi files

The "status" property was defined after the pllX subnodes
in pll node, which violates the Devicetree Specification
v0.4, section 6.3:

"Nodes may contain property definitions and/or child node
definitions. If both are present, properties shall come
before child nodes."

This caused the Device Tree Compiler error: "Properties
must precede subnodes. Unable to parse input tree"

This commit moves the "status" property to precede all pll subnodes,
fixing the syntax error and ensuring compliance with the Devicetree
Specification.

Signed-off-by: Stanislav Bobokalo <stas.gurland@gmail.com>
